src/main/java/seedu/command/DeleteCommand.java

```diff
@@ -29,7 +29,7 @@ public CommandResult execute() {
             int index = Integer.parseInt(strIndex) - 1;
             assert index < taskList.getListSize() : "index > the size of taskList";
             Task removedTask = taskList.deleteTask(index);
-
+            
```

Comment:
What was changed here from the original? Was the only change here the new line?

Reply:
Cause this master branch's method deleteCommand is different from my branch. So it may different